### DISCUSSION TITLE
How to add multiple GIT versions?

### DISCUSSION THREAD

**Discussion by MicheleDeF** [leokhoa/laragon#417] [General]
Hi, I'm using the laragon portable version, is it possible to install multiple GIT versions?

**Comment by Pen-y-Fan** [comment]
I don't know 🤷, you can try downloading the portable version and installing it in your laragon-portable/bin/git folder.

- [Download for Windows](https://git-scm.com/download/win) - try the Portable ("thumbdrive edition")

Let us know how you get on.

**Reply by MicheleDeF** [reply]
Hi, it worked, I replaced my old version of git installed on laragon portable with newer git version of type Portable. Thanks for your help.
